packages/shared/src/components/cards/list/CardCover.tsx

```diff
@@ -13,11 +13,16 @@ interface CardCoverProps extends ReusedCardCoverProps {
 
 export function CardCoverList({
   className,
+  imageProps,
   ...props
 }: CardCoverProps): ReactElement {
   return (
     <SharedCardCover
       {...props}
+      imageProps={{
+        ...imageProps,
+        className: classNames(imageProps.className, 'tablet:w-60'),
+      }}
```

Comment:
Fixed the inconsistent sizes of images which produces content shift.

Before:

https://github.com/user-attachments/assets/028d7ea5-c7e1-4cfd-984b-e772fe02ba2d

Figma:
![Screenshot 2024-07-19 at 11 08 46 PM](https://github.com/user-attachments/assets/855b3e85-edf9-43a4-ac3d-c2287eab8e80)

After:

![Screenshot 2024-07-19 at 11 09 33 PM](https://github.com/user-attachments/assets/fc7ec415-0244-47b6-b6fa-22f253c9ff90)